test/coverage.jl

```diff
@@ -13,7 +13,11 @@ struct Trash <: Rematch2.BoundFetchPattern
     location::LineNumberNode
 end
 
-expected="""
+@testset "Tests that add code coverage" begin
+
+@testset "Check the actual output of dumpall" begin
```

Comment:
indent the inner testsets for readability? (or at least comment the `end`s with what they match to)